app/controllers/stories_controller.rb

```diff
@@ -69,7 +69,7 @@ def assign_hero_html
 
   def get_latest_campaign_articles
     campaign_articles_scope = Article.tagged_with(SiteConfig.campaign_featured_tags, any: true).
-      where("published_at > ?", 4.weeks.ago).where(approved: true).
+      where("published_at > ? AND score > ?", 4.weeks.ago, 0).where(approved: SiteConfig.campaign_articles_require_approval?).
```

Comment:
Is it a problem that this check will filter out all posts that are explicitly approved when approval is not required?

The query will evaluate to `.where(approved: false)` which will filter out all explicitly approved posts. 

See other inline comment below